timescaledb/how-to-guides/page-index/page-index.js

```diff
@@ -308,8 +308,8 @@ module.exports = [
         href: "compression",
         children: [
           {
-            title: "Compression",
-            href: "compression"
+            title: "About compression",
+            href: "about-compression"
```

Comment:
A corresponding REDIRECT will be needed from old to new in the 301 redirect file.

Reply:
Oh, good idea. I'll do that now. I also searched the docs repo for instances of links and couldn't find any.

Reply:
https://github.com/timescale/web-documentation/pull/206